lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -25,6 +25,8 @@ def transform(evss_data)
         disabilities = form526['disabilities']
         lh_request_body.disabilities = transform_disabilities(disabilities)
 
+        direct_deposit = form526['directDeposit']
+        lh_request_body.direct_deposit = transform_direct_deposit(direct_deposit)
```

Comment:
for the missing `directDeposit` node, we can place the `if direct_deposit.present?` at the end of this line to check before adding to the request body

Reply:
@aurora-a-k-a-lightning fixed